src/map/lua/lua_baseentity.cpp

```diff
@@ -664,6 +664,28 @@ void CLuaBaseEntity::setVolatileCharVar(std::string const& varName, int32 value,
     }
 }
 
+/************************************************************************
+ *  Function: getAllLocalVars()
+ *  Purpose : Returns all variables assigned locally to an entity
+ *  Example :
+ *  Notes   :
+ ************************************************************************/
+
+auto CLuaBaseEntity::getAllLocalVars() -> sol::table
+{
+    auto                          table     = lua.create_table();
+    std::map<std::string, uint32> localVars = m_PBaseEntity->GetAllLocalVars();
```

Comment:
Use `auto&` here

Reply:
> Use `auto&` here

Thanks, I updated the PR, hopefully this is ok now.